src/http-spec.ts

```diff
@@ -24,6 +24,12 @@ export interface IHttpService extends INode {
     url?: string;
     identifier?: string;
   };
+  logo?: {
```

Comment:
hmm, which OAS (or other) part of the document are you going to consume to populate this data?

Reply:
This is populated by `"x-logo"`. I you think that the name is to general, I can specify it to `x-logo` instead of `logo`. I just wanted to make it more general in case we ever need to populate it from different source

Reply:
We want to support: https://github.com/Redocly/redoc/blob/master/docs/redoc-vendor-extensions.md#x-logo